test/avocado/selenium-login.py

```diff
@@ -32,101 +32,141 @@
 import inspect
 import selenium.webdriver
 from selenium.webdriver.common.desired_capabilities import DesiredCapabilities
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
 import os
+import sys
 import re
 import time
+machine_test_dir = os.path.dirname(
+    os.path.abspath(inspect.getfile(inspect.currentframe())))
+sys.path.append(machine_test_dir)
+import libdisc
 
 user = "test"
 passwd = "superhardpasswordtest5554"
+javascript_clicks=True
 
+present = EC.presence_of_element_located
```

Comment:
Why do we need aliases for these?

Also, at least `present` seems to be unused.